tools/shared.py

```diff
@@ -1700,7 +1700,12 @@ def make_fetch_worker(source_file, output_file):
     exit_with_error('emscripten config file not found: ' + CONFIG_FILE)
 
 parse_config_file()
-SPIDERMONKEY_ENGINE = fix_js_engine(SPIDERMONKEY_ENGINE, listify(SPIDERMONKEY_ENGINE))
+# Engine tweaks
+if SPIDERMONKEY_ENGINE:
+  new_spidermonkey = SPIDERMONKEY_ENGINE
+  if '-w' not in str(new_spidermonkey):
+    new_spidermonkey += ['-w']
```

Comment:
I think we can remove this soon. `-w` is used to get sm to print asm.js validation logging, but we don't need that once fastcomp is gone.

Reply:
Sure, I'll leave this change as NFC for now though I think.